test/jdk/com/sun/jdi/lib/jdb/Debuggee.java

```diff
@@ -106,7 +105,6 @@ public Launcher setSuspended(boolean value) {
         // required to pass non null port with address and emit string before the throw
         public Launcher enableOnThrow(String value, String expectedOutputBeforeThrow) {
```

Comment:
```suggestion
        public Launcher enableOnThrow(String exceptionClassName) {
```
expectedOutputBeforeThrow is not needed.
Comment for the method is obsolete